bigbluebutton-html5/imports/utils/debounce.js

```diff
@@ -0,0 +1,43 @@
+export function debounce(func, delay, options = {}) {
```

Comment:
Could you add a few lines of explanation for the function (including that `options` is comprised of `leading`, `trailing` etc)? I.e. lodash, radash, missing options...